lib/attr.gd

```diff
@@ -37,8 +37,14 @@
 ##
 ##  <Description>
 ##  This info class (together with <Ref InfoClass="InfoWarning"/> is used
-##  for messages about attribute storing  being  disabled  (at  level  2)  or
-##  enabled (level 3). It may be  used  in  the  future  for  other  messages
+##  for messages about attributes. The current warnings are:
+##  <List>
+##    <Item> Attribute storing  being  disabled  (level  2) </Item>
+##    <Item> Atribute storing being enabled (level 3)</Item>
```

Comment:
Again, I don't understand. When is this shown? Perhaps by attribute setters but when the previous message is not shown?!